pygmt/src/text.py

```diff
@@ -109,28 +109,28 @@ def text_(
     {J}
```

Comment:
Can't change it, but in line 82 the horizontal and vertical capital letters should be bolded. Additionally, it seems like the project uses American England, so it should read like:

```
        * Horizontal: **L**\ (eft), **C**\ (enter), **R**\ (ight)
        * Vertical: **T**\ (op), **M**\ (iddle), **B**\ (ottom)
```